src/import/chrome.ts

```diff
@@ -264,3 +265,15 @@ export function importFromChromeCPUProfile(chromeProfile: CPUProfile): Profile {
   profile.setValueFormatter(new TimeFormatter('microseconds'))
   return profile.build()
 }
+
+/**
```

Comment:
Thanks for writing this comment! I think this would more helpfully live at the top of `v8cpuFormatter.ts`.

I think this could also use a bit of rewording. Something like this:

```
* This importer handles an old format used by the C++ API of V8. This format is still used by v8-profiler-node8.
* There are two differences between the two formats:
*  - Nodes are a tree in the old format and a flat array in the new format
*  - Weights are timestamps in the old format and deltas in the new format.
*
* For more information, see https://github.com/hyj1991/v8-profiler-node8
```

Your English is still much, much better than my French 😄

Merci beaucoup pour tout votre contributions! 🇨🇦